addons/RB205_custom/5463_junior/config.cpp

```diff
@@ -0,0 +1,68 @@
+class cfgPatches
+{
+class RB205_custom_junior
+{
+    requiredAddons[] =
+    {
+        "RB205_main",
+        "RB205_custom"
+    };
+    requiredVersion = 1.0;
+    weapons[] =
+    {
+        "RB205_H_junior",
+        "RB205_U_junior"
+    };
+    units[]	=
+    {
+        "RB205_clone_junior"
+    };
+};
+};
+
+#include "\RB205_main\macros.hpp"
+
+class cfgWeapons
+{
+class RB205_H_arf_trooper;
+class RB205_H_junior: RB205_H_arf_trooper
+{
+    displayName = "[205] Clone ARF Trooper Helmet (5463)";
+    hiddenSelectionsTextures[] =
+    {
+        "RB205_custom\5463_junior\data\H_junior.paa",
+        "",
+        "RB205_main\data\arf\visor_arf_co.paa"
+    };
+};
+
+class RB205_U_arf_trooper;
+class ItemInfo;
+class RB205_U_junior: RB205_U_arf_trooper
+{
+    displayName = "[205] Clone ARF Trooper Uniform (5463)";
```

Comment:
Wir nennen die Uniform überall Armor und machen die ID außerdem in eckigen Klammern [ ] also:
"[205] Clone ARF Trooper Armor [5463]"